packages/engineer/src/feature/dev-server.dev-server.env.ts

```diff
@@ -143,12 +157,32 @@ devServerFeature.setup(
             await new Promise((resolve) => {
                 compiler.hooks.done.tap('compiled', resolve);
             });
+            if (serveExternalFeaturesPath) {
+                app.use('/plugins', express.static(resolve(externalFeaturesPath)));
```

Comment:
this is the path of the base location of the external features (node_modules for example)